### DISCUSSION TITLE
How to make Altitude Hold Autopilot? For f15-unreal engine 5

### DISCUSSION THREAD

**Discussion by buseeryas** [JSBSim-Team/jsbsim#1016] [Q&A]
How to make Altitude Hold Autopilot? For f15- unreal engine 5

**Comment by gallonmate** [comment]
There is no autopilot included in the model for the F15. You would have to add it yourself as seen in this discussion https://github.com/JSBSim-Team/jsbsim/discussions/444.
Although it looks like users still were having problems solving a few issues with the system.
